.github/workflows/projects-auto-move-new-sprint.yml

```diff
@@ -0,0 +1,21 @@
+name: Automatically Move Iteration Forward
+on:
+  schedule:
+    # Runs "at 05:00, only on Thursday" (see https://crontab.guru)
+    - cron: '0 5 * * 4'
+
+jobs:
+  move-to-next-iteration:
+    name: Move to next iteration
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: blombard/move-to-next-iteration@master
+      with:
+        owner: CDCgov
+        number: 33
+        token: ${{ secrets.PROJECTS_MOVE_TO_NEXT_ITERATION }}
+        iteration-field: Iteration
```

Comment:
I'm hoping this option is for naming the new sprint and all we have to do is rename it once this action happens but if this is suppose to be the name of the previous sprint then it might cause this action to fail. This public action lacks maturity and documentation which makes it hard to know.